ush/config_defaults.yaml

```diff
@@ -1980,91 +1982,49 @@ task_run_fcst:
-  #
-  # POST_OUTPUT_DOMAIN_NAME:
-  # Domain name (in lowercase) used to construct the names of the output files
-  # generated by the UPP. If using a predefined grid, this variable defaults to PREDEF_GRID_NAME. 
-  # 
-  # If using a custom grid, POST_OUTPUT_DOMAIN_NAME must be specified by 
-  # the user. Note that this variable is first changed to lowercase
-  # before being used to construct the file names.
-  # The post output files are named as follows:
-  # 
-  #   ${NET_default}.tHHz.[var_name].f###.${POST_OUTPUT_DOMAIN_NAME}.grib2
-  #
-  # TESTBED_FIELDS_FN
-  # The file that lists grib2 fields to be extracted for testbed files.
-  # An empty string means no need to generate testbed files.
-  #
-  # NUMX:
-  # The number of i regions in a 2D decomposition.  Each i row is 
-  # distibuted to numx ranks. Default value of numx is 1.
-  # Note that 2D decomposition does not yet work with GNU compilers, so this value 
-  # will be reset to 1 automatically when using GNU compilers (i.e., when COMPILER: gnu).
-  #
-  #-----------------------------------------------------------------------
-  #
-  USE_CUSTOM_POST_CONFIG_FILE: true
-  CUSTOM_POST_CONFIG_FP: "{{ user.SORCdir }}/ufs-weather-model/tests/parm/postxconfig-NT-fv3lam.txt"
-  POST_OUTPUT_DOMAIN_NAME: '{{ workflow.PREDEF_GRID_NAME }}'
-  TESTBED_FIELDS_FN: ""
-  NUMX: 1
+  upp: &det_upp
+    execution:
+      batchargs:
+        nodes: 1
+        tasks_per_node: 24
+        walltime: 00:15:00
+      envcmds:
+        - module load {{ user.HOMEdir }}/modulefiles
+        - module load build_{{ user.MACHINE|lower }}_{{ workflow.COMPILER }}
+      executable: '{{ user.EXECdir }}/upp.x'
+      mpicmd: '{{ platform.BATCH_RUN_CMD }}'
+      threads: 1
+    files_to_copy:
+      eta_micro_lookup.dat: '{{ user.PARMdir }}/upp/nam_micro_lookup.dat'
+      params_grib2_tbl_new: '{{ user.PARMdir }}/upp/params_grib2_tbl_new'
+      postxconfig-NT.txt: '{{ user.UFS_WTHR_MDL_DIR }}/tests/parm/postxconfig-NT-fv3lam.txt'
+    namelist:
+      update_values:
+        model_inputs:
+          datestr: "{{ timevars.valid_time }}"
+          filename: '{{ task_run_fcst.rundir }}/dynf{{ timevars.fff }}.nc'
+          filenameflux: '{{ task_run_fcst.rundir }}/phyf{{ timevars.fff }}.nc'
+          grib: grib2
+          ioform: netcdf
+          modelname: FV3R
+        nampgb:
+          kpo: 47
+          po: [1000.,975.,950.,925.,900.,875.,850.,825.,800.,775.,750.,725.,700.,675.,650.,625.,600.,575.,550.,525.,500.,475.,450.,425.,400.,375.,350.,325.,300.,275.,250.,225.,200.,175.,150.,125.,100.,70.,50.,30.,20.,10.,7.,5.,3.,2.,1.]
+    rundir: '{{ task_run_fcst.rundir }}/postprd/{{ timevars.fff }}'
+  platform:
+    account: '{{ user.ACCOUNT }}'
+    scheduler: '{{ platform.SCHED }}'
+  post_output_domain_name: '{{ workflow.PREDEF_GRID_NAME }}'
+  desired_output_name: '{{ nco.NET_default }}.{{ timevars.yyyymmddhh }}.{{ file_label }}.f{{ timevars.fff }}.{{ task_run_post.post_output_domain_name }}.grib2'
```

Comment:
`desired_output_name` was hard-coded in the scripts. It's factored out here to give users more control over what they'd like the name to be, and also for down-stream tasks to have access to it.